src/DynamicRandomVariableGenerator.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: BUSL 1.1
+// Valorem Labs Inc. (c) 2022.
+pragma solidity 0.8.16;
+
+import "./interfaces/IDynamicRandomVariableGenerator.sol";
+
+contract DynamicRandomVariableGenerator is IDynamicRandomVariableGenerator {
```

Comment:
I think we should be using a library here rather than a contract with interfaces.